hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/RSRpcServices.java

```diff
@@ -486,7 +488,9 @@ boolean incNextCallSeq(long currentSeq) {
     // cache the String once made.
     @Override
     public String toString() {
-      return this.clientIPAndPort + ", " + this.r.getRegionInfo().getRegionNameAsString();
+      return "clientIPAndPort=" + this.clientIPAndPort +
+        ", userName=" + this.userName +
+        ", regionInfo=" + this.r.getRegionInfo().getRegionNameAsString();
```

Comment:
I like this change but I think you should do the encoded regionname here rather than the full thing? The former is 32 bytes. The latter could be anything.  On other hand, full name would be more convenient. If only emitted on scan lease expiration, then maybe this is ok. What do you think?

Reply:
> I like this change but I think you should do the encoded regionname here rather than the full thing? The former is 32 bytes. The latter could be anything. On other hand, full name would be more convenient. If only emitted on scan lease expiration, then maybe this is ok. What do you think?

Thank you very much for your reply and suggestion. I think your suggestion is reasonable, and the full name may be quite irregular. But the full name does make it easy to trace, and I think it's probably better to use the full name here.